src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -383,10 +379,8 @@ export default class AircraftController {
             this._updateAircraftConflicts(aircraftModel, i);
             this._updateAircraftVisibility(aircraftModel);
 
-            // `#isFlightStripRemovable` will be true even when there is no corresponding
-            // `StripView` for and `aircraftModel`
-            if (aircraftModel.isFlightStripRemovable && this._stripViewController.hasStripViewModel(aircraftModel)) {
-                this._stripViewController.removeStripView(aircraftModel);
+            if (!aircraftModel.isControllable) {
```

Comment:
Good catch, I like this a ton! It was a poor decision to search twice like that 🤦.